docker-compose.yml

```diff
@@ -109,27 +119,42 @@ services:
     volumes:
```

Comment:
The command section is obsolete with vault  image itself. Please remove the command section in line 116-118

Reply:
removed.